packages/compass-e2e-tests/helpers/commands/wait-for-connection-result.ts

```diff
@@ -8,13 +9,18 @@ export async function waitForConnectionResult(
 ): Promise<undefined | string> {
   let selector: string;
   if (connectionStatus === 'either') {
+    // TODO: this doesn't support compass-web yet, but also isn't encountered yet
     // For the rare cases where we don't care whether it fails or succeeds
     selector = `${Selectors.DatabasesTable},${Selectors.ConnectionFormErrorMessage}`;
   } else if (connectionStatus === 'success') {
     // First meaningful thing on the screen after being connected, good enough
     // indicator that we are connected to the server
-    selector = Selectors.MyQueriesList;
+    // TODO: the web selector might actually work for both cases
+    selector = TEST_COMPASS_WEB
+      ? '[data-testid="workspace-tab-button"][title=Databases]'
+      : Selectors.MyQueriesList;
   } else {
+    // TODO: this doesn't support compass-web yet, but also isn't encountered yet
```

Comment:
Which TODOs are meant to be addressed in this PR and which are going to be part of follow-up work?